packages/core/content-manager/admin/tests/mockData.ts

```diff
@@ -371,6 +371,9 @@ const CM_CONTENT_TYPE_MOCK_DATA = [
       id: {
         type: 'string',
       },
+      name: {
+        type: 'string',
+      },
```

Comment:
ℹ️  name is returned in the mock server response but it is not defined on the schema so I added it